apps/openmw/mwgui/containeritemmodel.cpp

```diff
@@ -47,6 +53,29 @@ ContainerItemModel::ContainerItemModel (const MWWorld::Ptr& source)
     mItemSources.push_back(source);
 }
 
+bool ContainerItemModel::allowedToUseItem(const MWWorld::Ptr& target) const
```

Comment:
Should be as simple as ```isAllowedToUse(player, mItemSources[0])``` unless I'm missing something.

Reply:
Yes, it seems you are right.